@kiva/kv-components/vue/KvMap.vue

```diff
@@ -168,6 +131,7 @@ export default {
 		if (!this.mapLibreReady && !this.leafletReady) {
 			this.initializeMap();
 		}
+		this.getLibraries();
```

Comment:
I think this call can go alongside `initializeMap` within the check above. We don't want to load the libraries multiple times...

We might also consider calling it within each block of the initializeMap method which will have established webGL support status by then and Also start loading the libarries before the checks start.

Reply:
Nice call, thanks!